### PR TITLE
Updating topic description

### PR DESCRIPTION
**Pull request by nurih**
Current description is terse and touches only on a minor aspect - that it runs on multiple platforms. A description which touches on the capabilities and nature of the database should be more relevant to newcomers.

### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection

---------------------------------------------------------------------
### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [x] Content (and my changes are in `index.md`)

> The existing short description mentions a minor attribute of the database only - that it can run on several platforms. My suggested edits highlight the nature of the database, making it more relevant to newcomers stumbling upon this topic.
---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by Saturnre** [comment]
I followed the contributing guidelines 
https://github.com/github/explore/blob/master/CONTRIBUTING.md
I have  no affiliation with the project I'm making a pull request as a software architect  Bitcoin Gold miner or suggesting edits and mergers to blockchains existing and future topics or collections.

File: topics/mongodb/index.md [hunk 1]
```diff
@@ -5,9 +5,9 @@ display_name: MongoDB
 github_url: https://github.com/mongodb
 logo: mongodb.png
 released: February 11, 2009
-short_description: MongoDB is a NoSQL cross-platform database.
+short_description: MongoDB is an open source NoSQL document-oriented database management system.
```
Comment (kenyonj) [inline topics/mongodb/index.md]: Thoughts on removing "management system" ? I think of MongoDB as a database and something like AWS Management Console a management system.

File: topics/mongodb/index.md [hunk 2]
```diff
@@ -5,9 +5,9 @@ display_name: MongoDB
...
 topic: mongodb
 url: https://www.mongodb.com/
 wikipedia_url: https://en.wikipedia.org/wiki/MongoDB
 ---
-MongoDB is a NoSQL cross-platform database. It is designed for scalability and performance.
+MongoDB is an open source NoSQL document-oriented database management system. MongoDB allows flexible schema representation instead of relying on fixed tables and columns. It supports full indexing, query, aggregation, tunable consistency and durability levels, and linear horizontal scaling through sharding.
```
Comment (kenyonj) [inline topics/mongodb/index.md]: same comment here, re: "management system"

File: topics/mongodb/index.md (hunk 1 above)
Comment (nurih) [reply topics/mongodb/index.md]: Good point, though MySQL and Postgress  include the terminology, and they are not a management console either. The term is commonly used in "traditional" database systems, giving birth to the acronym RDBMS.

Comment (kenyonj) [reply topics/mongodb/index.md]: Thanks for the explanation, TIL! 

With that, this LGTM.

**Comment by MikeMcQuaid** [comment]
Thanks so much for your first contribution! Without people like you submitting PRs we couldn't run this project. You rock, @nurih!

**Pull request merged**